src/FlapperUniV2.sol

```diff
@@ -35,7 +35,6 @@ interface SpotterLike {
 interface GemLike {
     function decimals() external view returns (uint8);
     function balanceOf(address) external view returns (uint256);
-    function approve(address, uint256) external;
```

Comment:
If we want to remove unused staff then we can also remove the `live()` from `VatLike`.
I'm not 100% sure these are worth changing that contract and differing from the deployed version.